site/package.json

```diff
@@ -13,10 +13,15 @@
     },
     "dependencies": {
         "@create-disruptions-data/shared-ts": "file:../shared-ts",
+        "@emotion/react": "^11.10.6",
```

Comment:
Are these emotion libraries needed? Seems to work ok for me without them

Reply:
Yeah it says in the docs they're needed. It works without them... for a while, and every now and again it errors (when running locally). I initially didnt have them, but it error'd every now and again.